andela/travel_tool_front
# #161889373 Set name of deleted travel team member in toast message

#### What does this PR do?
This PR adds deleted travel team member's name to the toast message on successful delete

#### Description of Task to be completed?
- Update delete travel team success message to contain deleted travel team member's name
- Update tests according to implementation


#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the`DB` and on the `UserRoles` table, set yourself as a `Travel Admin`
- Go to the browser, refresh the page, Navigate to the `Settings` tab and  click on `Travel Team`
- Click `Add user` to add travel team member if non currently exist.
- Click the `delete` button on a member's role
- A `confirmation modal` will pop up
- `confirm delete`
- The deleted member's name should appear on the success toast message: `<deleted member's name> removed successfully`

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[#161889373](https://www.pivotaltracker.com/story/show/161889373)

#### Screenshots (if appropriate)
![delete](https://user-images.githubusercontent.com/27569442/48418088-b3866a80-e754-11e8-8755-48871164c0be.gif)


#### Questions:
N/A
